LBH.AdultSocialCare.Api/V1/Domain/DayCarePackageDomains/DayCarePackageDomain.cs

```diff
@@ -23,9 +23,6 @@ public class DayCarePackageDomain
         public bool TransportNeeded { get; set; }
         public bool EscortNeeded { get; set; }
         public int TermTimeConsiderationOptionId { get; set; }
-        public string HowLong { get; set; }
-        public string HowManyTimesPerMonth { get; set; } // Daily, weekly, monthly
-        public string OpportunitiesNeedToAddress { get; set; }
         public DateTimeOffset DateCreated { get; set; } = DateTimeOffset.UtcNow;
```

Comment:
Just for future reference - we should no longer need the date created / date updated fields on our objects as they should now be handled by BaseEntity